pages/2018/index.rst

```diff
@@ -38,9 +38,97 @@ All participants at the conference will be expected to follow the `code of condu
+|3:00pm   |                       |                       |Summary of        |Coffee           |Talk (Daniel   |
+|         |                       |                       |unconferences     |                 |Foreman-Mackey)|
++---------+-----------------------+-----------------------+------------------+-----------------+               |
+|3:30pm   |Coffee                 |Coffee                 |Hack day          |Hacking          |               |
+|         |                       |                       |pre-pitches       |                 |               |
++---------+-----------------------+-----------------------+------------------+                 |               |
+|4:00pm   |Summary of             |Summary of             |Public talk       |                 |               |
+|         |unconferences          |unconferences          |reception -       |                 |               |
++---------+-----------------------+-----------------------+tea/snacks        |                 |               |
+|4:30pm   |Lightning talks        |Lightning talks        |                  |                 |               |
+|         |                       |                       |                  |                 |               |
++---------+                       |                       +------------------+-----------------+---------------+
+|5:00pm   |                       |                       |Public talk       |Hack             |Wine and       |
+|         |                       |                       |(Kelle Cruz)      |check-in         |cheese         |
++---------+-----------------------+-----------------------+                  |                 +---------------+
+|5:30pm   |                       |                       |                  |                 |               |
+|         |                       |                       |                  |                 |               |
++---------+-----------------------+-----------------------+                  +-----------------+---------------+
+|6:00pm   |Wine and cheese        |                       |                  |Conference       |               |
+|         |                       |                       |                  |dinner           |               |
++---------+-----------------------+-----------------------+------------------+-----------------+---------------+
+
+..
+   The full list of talk titles and abstracts can be found here. **PUT A LINK HERE**
+
+Below are some notes on some important aspects of the conference program:
+
+*Unconference sessions*
+  Unconference and tutorial slots will include some planned sessions but attendees are encouraged to propose their own sessions during the pitches.
+  These will run in parallel with the pre-organised sessions.
+
+*Lightning talks*
+  These will provide an opportunity for attendees to make short (~5 minutes) presentations on any topic.
```

Comment:
Let's say 3-5 rather than ~5? I would rather give everyone the chance to have a shorter talk than just an opportunity for a subset of attendees.